packages/synapse-react-client/src/components/Plot/PlotPanelHeader.tsx

```diff
@@ -9,19 +9,18 @@ import { IconSvgButton } from '../IconSvgButton'
 
 export type PlotPanelHeaderProps = {
   data?: QueryResultBundle
-  isLoadingNewBundle?: boolean
+  isLoading?: boolean
```

Comment:
Rename so it's clear that this just makes the component show loading UI. `isLoadingNewBundle` was not helpful as a prop here